packages/e2e/src/chopsticks-provider.test.ts

```diff
@@ -0,0 +1,99 @@
+import { ApiPromise } from '@polkadot/api'
+import { ChopsticksProvider } from '@acala-network/chopsticks-core'
+import { afterAll, beforeAll, describe, expect, it } from 'vitest'
+
+import { env, expectHex, expectJson, mockCallback, testingPairs } from './helper'
+
+// TODO: to be enabled after impl worker thread for nodejs compatibility
```

Comment:
write a followup issue if you don't plan address this in this PR and put the issue number here so we don't forget to do it in future